### PR TITLE
Update tokens in mainnet

### PR DESCRIPTION
**Pull request by s-medvedev**

### CODE REVIEW

File: tokenlist.json [hunk 1]
```diff
@@ -323,96 +323,59 @@
...
-      "chainId": 245022934,
-      "address_spl": "ApmXkxXCASdxRf3Ln6Ni7oAZ7E6CX1CcJAD8A5qBdhSm",
-      "address": "0x633AC0dE8749f6c2fACb73665a05c622a272f291",
-      "decimals": 9,
-      "name": "Curve DAO Token (Wormhole)",
-      "symbol": "wCRV",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/curve-dao-logo.svg"
+      "name": "Wrapped Ether (Wormhole)",
+      "symbol": "WETH",
+      "logoURI": ""
```
Comment (s-medvedev) [inline tokenlist.json]: @oable please, provide logoURL

File: tokenlist.json [hunk 2]
```diff
@@ -323,96 +323,59 @@
...
-      "address": "0xa2b3D9E1E92Af208Ed6bc41d4428e500569Af66F",
+      "address_spl": "9n4nbM75f5Ui33ZbPYXn59EwSgE8CGsHtAeTH5YFeJ9E",
+      "address": "0x54EcEC9D995A6CbFF3838F6a8F38099E518805d7",
       "decimals": 6,
-      "name": "Wrapped SUSHI (Sollet)",
-      "symbol": "SUSHI",
-      "logoURI": "https://raw.githubusercontent.com/neonlabsorg/token-list/master/assets/sushiswap-sushi-logo.svg"
+      "name": "Wrapped Bitcoin (Sollet)",
+      "symbol": "BTC",
+      "logoURI": ""
```
Comment (s-medvedev) [inline tokenlist.json]: @oable please, provide logoURL

Comment (oable) [reply tokenlist.json]: I added icon. Will available by link after merging to main branch.

File: tokenlist.json (hunk 1 above)
Comment (oable) [reply tokenlist.json]: I added icon. Will available by link after merging to main branch.

**Pull request merged**